src/VisualStudio/Core/Impl/CodeModel/AbstractProjectCodeModel.cs

```diff
@@ -1,86 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using Microsoft.VisualStudio.LanguageServices.Implementation.Interop;
-using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
-using Roslyn.Utilities;
-
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeModel
-{
-    internal abstract class AbstractProjectCodeModel
```

Comment:
Note this moved to ProjectCodeModel.cs; GitHub isn't catching the rename.

Reply:
Seems fine now.

Reply:
@CyrusNajmabadi Probably fine if you do commit-by-commit; it's broken if you view PR as a whole.